students/jeonghyeon/users/vaildation.py

```diff
@@ -0,0 +1,15 @@
+import re
+from django.forms import ValidationError
+
+
+REGEX_PASSWORD ='^(?=.*[A-Za-z])(?=.*\d)(?=.*[@$!%*#?&])[A-Za-z\d@$!%*#?&]{8,}$'
+REGEX_EMAIL = '^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$'
+
+
+def Vaildation_email(email):
```

Comment:
함수는 동사이며 대문자를 쓰지 않습니다.
```suggestion
def vaildate_email(email):
```

Reply:
소문자 수정완료